setup.cfg

```diff
@@ -6,17 +6,31 @@ summary = Simple dataset to dataloader library for pytorch
 description-file =
     README.rst
 description-content-type = text/x-rst; charset=UTF-8
+home-page = https://github.com/aiwizo/pytorch-datastream
 project_urls =
-    Source Code = https://github.com/aiwizo/pytorch-datastream/
+    Source Code = https://github.com/aiwizo/pytorch-datastream
+    Bug Tracker = https://github.com/aiwizo/pytorch-datastream/issues
 license = Apache-2
+license_files = LICENSE
+platform = any
```

Comment:
Haven't tested that this works on windows and I hope we don't intend to spend time supporting windows if there is an issue

Reply:
you are correct, perhaps we should be a bit more cautious and add Windows when we have people asking for it?

Reply:
Yes, although I am of the opinion to never support windows :)